overlord/devicestate/devicemgr.go

```diff
@@ -989,6 +989,55 @@ func (m *DeviceManager) ensureSeeded() error {
 	return nil
 }
 
+var processAutoImportAssertionsImp = processAutoImportAssertions
+
+// ensureAutoImportAssertions makes sure that auto import assertions
+// get processed. assertion should be processed while seeding is in progress
+func (m *DeviceManager) ensureAutoImportAssertions() {
+	if release.OnClassic {
+		return
+	}
+
+	if m.earlyDeviceSeed == nil {
+		// we have no seed cached yet, no point to check further
+		return
+	}
+
+	m.state.Lock()
+	defer m.state.Unlock()
+
+	if !m.changeInFlight("seed") {
```

Comment:
this is a bit odd, shouldn't it be a check simply that we are not seed yet?

Reply:
I have changed this and I'm checking if system is seeded then I stop trying.